pkgs/development/python-modules/fastembed/default.nix

```diff
@@ -26,9 +30,13 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [
     poetry-core
+    pythonRelaxDepsHook
+    setuptools
```

Comment:
Not sure why you're adding setuptools.

https://github.com/qdrant/fastembed/blob/v0.1.2/pyproject.toml#L35-L37